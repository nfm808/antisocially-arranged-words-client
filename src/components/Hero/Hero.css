.Hero {
  height: 100vh;
  display: flex;
  flex-direction: column;
  align-content: center;
  justify-content: center;
}

.Hero--text {
  padding: 20px;
}