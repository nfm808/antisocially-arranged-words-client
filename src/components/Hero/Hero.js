import React from 'react';
import PropTypes from 'prop-types';
import './Hero.css';

export default function Hero(props) {
  return (
    <section className='Hero'>
      <div className='Hero--text'>
        <h1>{props.heading}</h1>
        <h3>{props.subHeading}</h3>
      </div>
    </section>
  )
}

Hero.propTypes = {
  heading: PropTypes.string,
  subHeading: PropTypes.string
}
