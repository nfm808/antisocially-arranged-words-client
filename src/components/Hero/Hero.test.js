import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Hero from './Hero';

describe('<Hero />', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Hero />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})