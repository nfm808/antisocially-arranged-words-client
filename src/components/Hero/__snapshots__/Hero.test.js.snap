// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> renders without crashing 1`] = `
<section
  className="Hero"
>
  <div
    className="Hero--text"
  >
    <h1 />
    <h3 />
  </div>
</section>
`;
